addon/webextension/selector/uicontrol.js

```diff
@@ -353,8 +353,9 @@ window.uicontrol = (function () {
       this.cachedEl = null;
       ui.Box.remove();
       ui.WholePageOverlay.display(standardOverlayCallbacks);
-      document.addEventListener("keyup", watchFunction(keyupHandler), false);
```

Comment:
this wasn't getting removed in `removeHandlers` since the wrapped function isn't the same as `keyupHandler`